src/duct/module/web.clj

```diff
@@ -80,6 +81,8 @@
 
 (def ^:private api-config
   {:duct.handler.static/bad-request           {:body ^:displace {:error :bad-request}}
+   :duct.handler.static/bad-request-malformed {:body "{\"error\": \"Malformed request body\"}"
```

Comment:
Hardcoded JSON, I don't want to add another JSON library as dependency.

Reply:
Since all the other handlers are data, rather than hardcoded, I don't think we should hardcode the JSON. Can we let Muuntaja handle the formatting of the error message?

Reply:
Probably yes, Muuntaja uses CHeshire itself.